examples/demo13.py

```diff
@@ -352,6 +352,10 @@ def main(argv):
         # First, we include the expected Poisson noise:
         final_image.addNoise(poisson_noise)
 
+        # At this point in the image generation process, an integer number of photons gets
+        # detected, hence we have to round the pixel values to integers:
+        final_image.quantize()
```

Comment:
I believe at this point, the image is already quantized unless any of the pre-noise values were > 2^30.  That's when our Poisson implementation switches over to the Gaussian approximation, which won't necessarily produce integers.
So probably we don't need this line.  Or if you want to keep it, maybe add some commentary about how it's not really necessary for non-saturated pixel values.

Reply:
Yes - I confirmed this directly as well, so have removed this step and added a comment.